contributing/new-page-template.md

```diff
@@ -0,0 +1,55 @@
+---
+# This section is called the "frontmatter" for your page
+title: "Title For Your Page" # Use title case for titles
+#linkTitle: "Link For This Page in the Sidebar"
+# The linkTitle field (above) is optional; use it to provide a shorter link if your page title is very long
+weight: 01 # This affects the placement of the link in the sidebar on the left. Pages are ordered from top to bottom by weight.
+type: "docs" # You won't need to update this.
+aliases:
+  - # Has the page ever moved? If yes, include the prior location here, starting with /docs/, and the old URL will redirect to the new location. For a new page, there should be no aliases.
+---
+
+This guide shows you how to do something very cool. Make sure to include
+a value proposition for the user: for example, this guide shows you how to do X,
+which is useful for doing Y and Z. Make sure you answer the questions "what does
+this guide show you how to do?" and "why would someone want to do this?".
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative installed](../install/README.md). <!-- Update this relative link as needed,
+depending on where the new page is located in the file structure. -->
+- Anything else?
+
+## Break steps into logical sections
+
+Avoid nesting headings directly on top of each other with no text inbetween.
+
+1. Use ordered lists for steps.
```

Comment:
Include a nested code example and highlight the importance of using spaces for Indentation (not tabs)